filebeat/publisher/sync.go

```diff
@@ -53,26 +54,26 @@ func (p *syncLogPublisher) Publish() error {
 	var events []*input.Event
 	select {
 	case <-p.done:
-		return errors.New("publishing was stopped")
+		return sigPublisherStop
 	case events = <-p.in:
 	}
 
 	ok := p.client.PublishEvents(getDataEvents(events), publisher.Sync, publisher.Guaranteed)
 	if !ok {
 		// PublishEvents will only returns false, if p.client has been closed.
-		return errors.New("publisher didn't published events")
+		return sigPublisherStop
```

Comment:
Should we really have the same signal here? Could ti be for other reasons that ok wasn't ture?